ggdp/assets.py

```diff
@@ -123,3 +125,23 @@ def raw_round_contributors() -> pd.DataFrame:
     df = round_file_aggregator("contributors.json")
     df["roundId"] = '"' + df["roundId"] + '"'
     return df
+
+@asset(
+    description="Metadata for chains on which Gitcoin indexer registered at least one round. Source: `chainid.network/chains.json`"
```

Comment:
Nitpick: Dagster will pick up the description from the Python function docstrings, making it more flexible as IDEs will also rely on the docstrings.